Makefile

```diff
@@ -90,14 +91,22 @@ cross-build: cross-build-linux-amd64 cross-build-linux-arm64
 _build_containerized: genbpfassets tidy-vendor format
 	@if [ -z '$(CTR_CMD)' ] ; then echo '!! ERROR: containerized builds require podman||docker CLI, none found $$PATH' >&2 && exit 1; fi
 	echo BIN_TIMESTAMP==$(BIN_TIMESTAMP)
+
+	rm -f $(SRC_ROOT)/build/Dockerfile.tmp
+	cp $(SRC_ROOT)/build/Dockerfile $(SRC_ROOT)/build/Dockerfile.tmp
+
+	@if [ '$(ROOTLESS)' = 'false' ] ; then echo "user root" >> $(SRC_ROOT)/build/Dockerfile.tmp; fi
```

Comment:
we don't need to add `user root`. If no user is defined, the default is then root.

Reply:
ok, updated